docs/operating-eck/uninstalling-eck.asciidoc

```diff
@@ -7,18 +7,20 @@ endif::[]
 [id="{p}-{page_id}"]
 = Uninstall ECK
 
-Before uninstalling the operator, remove all Elastic resources in all namespaces:
+To uninstall the operator:
 
+. Remove all Elastic resources in all namespaces:
++
 [source,shell]
 ----
 kubectl get namespaces --no-headers -o custom-columns=:metadata.name \
   | xargs -n1 kubectl delete elastic --all -n
 ----
++
+This deletes all underlying Elasticsearch, Kibana, and APM Server resources like Pods, Secrets, Services, and so on.
```

Comment:
We also have `Enterprise Search` and `Beats` now.
Maybe a generic `This deletes all underlying Elastic stack resources and their Pods, Secrets, Service, and so on` would work?